sdk/spring/azure-spring-boot-samples/azure-spring-boot-sample-active-directory-b2c-oidc/README.md

```diff
@@ -24,10 +24,14 @@ Follow the guide of [AAD B2C user flows creation](https://docs.microsoft.com/azu
 ## Examples
 ### Configure the sample
 
-#### Application.yml
+#### application.yml
+
+1. Fill in `${your-tenant-endpoint-base-uri}` from **Azure AD B2C** portal `App registrations` blade, select **Endpoints**, copy the base endpoint uri(Global cloud format may looks like
+`https://{your-tenant-name}.b2clogin.com/{your-tenant-name}.onmicrosoft.com`, China Cloud looks like `https://{your-tenant-name}.b2clogin.cn/{your-tenant-name}.partner.onmschina.cn`). 
+
+    **NOTE**: If you want to continue to use the `tenant` configuration, please note this is only for Global cloud, fill in `${your-tenant-name}` from **Azure AD B2C** portal `Overviews` domain name (format may looks like
+    `${your-tenant-name}.onmicrosoft.com`).
```

Comment:
How about change to this:
The `azure.activedirectory.b2c.tenant` has been deprecated. Please use `azure.activedirectory.b2c.base-uri` instead.